test/controllers/api/v2/repositories_controller_test.rb

```diff
@@ -142,6 +142,20 @@ def test_index_with_content_view_version_id
       assert_template 'api/v2/repositories/index'
     end
 
+    def test_index_with_content_view_version_id_and_library
+      ids = @view.versions.first.repositories.pluck(:library_instance_id).reject(&:blank?)
+
+      @controller
+      .expects(:item_search)
+      .with(anything, anything, has_entry(:filters => [{:terms => {:id => ids}}]))
+      .returns({})
```

Comment:
Nitpick -- since these are part of the `@controller` they should be indented to indicate they are a continuance and not stand alone.